core/play-guice/src/test/scala/play/api/libs/akka/AkkaGuiceSupportSpec.scala

```
@@ -0,0 +1,124 @@
+/*
+ * Copyright (C) 2009-2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package play.api.libs.akka
+
+import javax.inject.{ Inject, Singleton }
+import akka.actor.typed.ActorRef
+import com.google.inject.AbstractModule
+import org.specs2.mutable.Specification
+import play.api.Configuration
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.libs.concurrent.AkkaGuiceSupport
+
+object AkkaGuiceSupportSpec extends Specification {
+  "AkkaGuiceSupport".title
+
+  "Runtime DI support for FP-style typed actors" in (fpStyle.getMain() !== null)
```

I'm pretty sure that, in the case of a failure, an exception will be thrown instead of returning null.